README.md

```diff
@@ -2,88 +2,65 @@
+* [Project Structure](#project-structure)
 * [Getting Started](#getting-started)
-* [Bug Reporting](#bug-reporting)
+* [Build Configs](docs/build-configs.md)
 * [Further Reading](#further-reading)
 * [Glossary](https://wiki.integreat-app.de/glossary)
 
 ## Contributing
-You can contribute by:
-* [Creating Pull requests](../.github/CONTRIBUTING.md#pull-requests)
-* [Reporting bugs](#bug-reporting)
 
-**Testing with the live cms instance should be avoided. Instead, [please use the test cms](docs/technology-stack.md#test-cms).**
+You can contribute by:
+* [Creating Pull requests](docs/contributing.md#pull-requests)
+* [Reporting bugs](docs/contributing.md#bug-reporting)
 
 If you want to know more about Integreat or if you want to join us, contact [Max](mailto:ammann@integreat-app.de),
 [Steffen](mailto:kleinle@integreat-app.de) or [our info mail](mailto:info@integreat-app.de) and visit our [website](https://integreat-app.de).
 
+## Project Structure
+
+This is a monorepo and contains the following projects:
+
+* **[web](web/README.md)**: React JS webapp
+* **[native](native/README.md)**: React Native Android and iOS apps
+
+Both projects are whitelabelled with different [build configs](docs/build-configs.md).
+    
+There are also separate sub-projects used by both projects:
+    
+* **[locales](locales/README.md)**: Localized texts and utilities to manage them
+* **[api-client](api-client/README.md)**: TODO: Not yet part of this repository.
+
 ## Getting Started
+
 ### Prerequisites
+
 * Rate our Integreat app in the [PlayStore](https://play.google.com/store/apps/details?id=tuerantuer.app.integreat)
 and the [Apple App Store](https://apps.apple.com/ae/app/integreat/id1072353915).
 * Install [nodejs](https://nodejs.org/). At least v10 is required, but we recommend the v12 LTS.
 Using the latest version (v13) may lead to errors.
-* Install [yarn](https://yarnpkg.com/)
-* For native development also take a look at the [Android setup](#android) and the [IOS setup](#ios).
-
-#### Android
-
-* Install the Android SDK by using the [Android Support plugin](https://plugins.jetbrains.com/plugin/1792-android-support) in IntelliJ.
-* Install the latest stable SDK Platform and Android SDK Tools in the SDK Manager (Settings > Appearance & Behaviour > System Settings > Android SDK)
-* \[optional\]: If you want to develop using an emulator, also install the Android Emulator in the Android SDK settings
-
-#### iOS
-
-* Own a Mac or another Apple device.
-* Install [XCode](https://developer.apple.com/xcode/).
-
-*Note: In order to work with the XCode project, always open `ios/Integreat.xcworkspace`.*
+* Install [yarn](https://yarnpkg.com/).
 
 ### Project setup
+
 We suggest **[IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/)** as IDE. JetBrains provides free licences for students.
 If you are using a different IDE, the steps might differ.
 
 * Import this project (VCS > Get from Version Control).
-* Run `yarn` in the terminal to install all dependencies.
```

Comment:
If you remove this part it needs to be mentioned that the complete setup differs depends on the projet, e.g. native or web or sth like that.

Reply:
Is the following not enough? `* Follow the steps mentioned in the projects README.`

Reply:
Sure

Reply:
What do you mean by sure? :D If you tell me what to write I'll include it :sunglasses:

Reply:
Oh wow. Somehow I kind of missed that. Sorry. You did well 👍🏼